avocado-plugins-vt.spec

```diff
@@ -21,7 +21,7 @@ Requires: python-imaging
 %if 0%{?el6}
 Requires: gstreamer-python, gstreamer-plugins-good, python-simplejson
 %else
-Requires: pygobject2, gstreamer1-plugins-good
+Requires: pygobject2, gstreamer1-plugins-good, python2-simplejson
```

Comment:
@xutian according to these packaging guidelines (under section Provides):

http://fedoraproject.org/wiki/Packaging:Python

It's better to stick to `python-simplejson`, which will be provided by `python2-simplejson`:

```
rpm -q --provides python2-simplejson 
python-simplejson = 3.10.0-1.el7
python-simplejson(x86-64) = 3.10.0-1.el7
python2-simplejson = 3.10.0-1.el7
python2-simplejson(x86-64) = 3.10.0-1.el7
```

With that, it's possible to se the same python-simplejson dep for all distros:

```diff
diff --git a/avocado-plugins-vt.spec b/avocado-plugins-vt.spec
index 4dd9eda..fddbb67 100644
--- a/avocado-plugins-vt.spec
+++ b/avocado-plugins-vt.spec
@@ -15,11 +15,11 @@ Source0: https://github.com/avocado-framework/%{modulename}/archive/%{commit}/%{
 BuildRequires: python2-devel, python-setuptools
 BuildArch: noarch
 Requires: avocado >= 36.3
-Requires: python, autotest-framework, p7zip, tcpdump, iproute, iputils, gcc, glibc-headers, python-devel, nc, aexpect, git, python-netaddr, python-netifaces
+Requires: python, autotest-framework, p7zip, tcpdump, iproute, iputils, gcc, glibc-headers, python-devel, nc, aexpect, git, python-netaddr, python-netifaces, python-simplejson
 
 Requires: python-imaging
 %if 0%{?el6}
-Requires: gstreamer-python, gstreamer-plugins-good, python-simplejson
+Requires: gstreamer-python, gstreamer-plugins-good
 %else
 Requires: pygobject2, gstreamer1-plugins-good
 %endif

```